src/kms_sink.cpp

```diff
@@ -141,6 +141,9 @@ int KMSSink::configurePipeline(const libcamera::PixelFormat& format) {
     case libcamera::formats::RGBA8888:
       xFormat = libcamera::formats::RGBX8888;
       break;
+    default: 
```

Comment:
Could this just be:

```
default:
  break;
```

Other than that this PR looks great! Most commonly at the moment, the format coming into this switch statement would be neither of the 4 explicit options, which is perfectly fine, don't want to to print out stuff to standard error, when there is no problem.